### PR TITLE
AG-8051 Footnote docs

### PR DESCRIPTION
**Pull request by sryazanov**
https://ag-grid.atlassian.net/browse/AG-8051

Add footnote option documentation.

### CODE REVIEW

**Review by lsjroberts — APPROVED** [review]
LGTM

File: grid-packages/ag-grid-enterprise/src/main.ts [hunk 1]
```diff
@@ -395,20 +395,16 @@ IsColumnFuncParams,
 ISelectCellEditorParams,
 ISelectionHandle,
 ISelectionHandleFactory,
-ISelectionService,
```
Comment (alantreadway) [inline grid-packages/ag-grid-enterprise/src/main.ts:398]: These should be unnecessary, head of `latest` is accurate right now (I added a CI check to detect this case a little while ago)

**Review by alantreadway — COMMENTED** [review]
Could you check if the updated image (removing series labels) is consistent with the documentation around it?

https://ag-grid.com/javascript-charts/getting-started/#labels-and-titles

^ it looks like the series labels should be present in the image?

File: grid-packages/ag-grid-enterprise/src/main.ts (hunk 1 above)
Comment (sryazanov) [reply grid-packages/ag-grid-enterprise/src/main.ts]: Reverted

**Comment by sryazanov** [comment]
Updated the two other images for consistency.

**Comment by sryazanov** [comment]
As David suggested I reversed the order of the series to make the legend appear in the increasing order. But it also reversed the stacks.

**Comment by alantreadway** [comment]
Do there need to be additions in the layout page for the new element in the diagram?

**Pull request merged**